### DISCUSSION TITLE
Next release?

### DISCUSSION THREAD

**Discussion by jana-starkova** [MrPowers/chispa#134] [General]
Hi @MrPowers, do you have a timeline for the next release? 🙏 My PR #133 has just been merged, so I am curious when I can expect to see this in prod. For now, I am using a workaround that removes column metadata which is a bit inconvenient. Thanks!

**Comment by MrPowers** [comment]
@jana-starkova - thanks for following up.  We've made a bunch of improvements recently.  Let's make a new release soon!
